src/screens/ui/InProgressActivityScreen.tsx

```diff
@@ -3,13 +3,15 @@ import { FC } from 'react';
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
 
 import { FlowSurvey } from '@app/widgets/survey';
+import { useUpcomingNotificationsObserver } from '@entities/notification';
 import { RootStackParamList } from '@screens/config';
 import { Box } from '@shared/ui';
 
 type Props = NativeStackScreenProps<RootStackParamList, 'InProgressActivity'>;
 
 const InProgressActivityScreen: FC<Props> = ({ navigation, route }) => {
   const { appletId, eventId, entityId, entityType } = route.params;
+  useUpcomingNotificationsObserver(eventId, entityId);
```

Comment:
If it's not a problem, could you add an empty line before useUpcomingNotificationsObserver?